src/apps/bridge/reportBuilder.cpp

```diff
@@ -55,6 +57,8 @@ static aanderaa_aggregations_t NAN_AGG = {.abs_speed_mean_cm_s = NAN,
                                           .std_tilt_mean_rad = NAN,
                                           .reading_count = 0};
 
+static soft_aggregations_t SOFT_NAN_AGG = {.temp_mean_deg_c = NAN, .reading_count = 0};
+
```

Comment:
the reading count is not sent out, but is part of the struct so it can be logged in the softSensor.cpp. (like many other spots, I think this can be decided on when doing the shared logs PR)